client/src/components/EventPage/index.jsx

```diff
@@ -42,40 +42,39 @@ class EventPage extends Component {
     }
   }
 
-  SubscribeEvent = (eve) => {
+  subscribeEvent = (eve) => {
     eve.preventDefault()
 
-    let data;
+    let data = {
+      'event_id': this.props.match.params.id,
+      'user_id': sessionStorage['User-id'],
+      'if_subbed': null
```

Comment:
You can set 'if_subbed' true by default. After this, you don't need an else statement.